releases/release-4.0.7.md

```diff
@@ -0,0 +1,100 @@
+---
+title: TiDB 4.0.7 Release Notes
+---
+
+# TiDB 4.0.7 Release Notes
+
+Release date: September 29, 2020
+
+TiDB version: 4.0.7
+
+## Improvements
+
++ TiDB
+
+    - Add more runtime information for join executor [#20093](https://github.com/pingcap/tidb/pull/20093)
+    - Add coprocessor cache hit ratio in explain analyze [#19972](https://github.com/pingcap/tidb/pull/19972)
+    - Support push ROUND function to TiFlash [#19967](https://github.com/pingcap/tidb/pull/19967)
+    - Add default value of CMSketch for Analyze [#19927](https://github.com/pingcap/tidb/pull/19927)
+
++ TiKV
+
+    - Support JSON log format [#8382](https://github.com/tikv/tikv/pull/8382)
+
```

Comment:
```suggestion

+ PD

    - Make counter inc when operator finished rather than operator added [#2983](https://github.com/pingcap/pd/pull/2983)
    - Set the `make-up-replica` operator to high priority [#2977](https://github.com/pingcap/pd/pull/2977)

```